tech_foul_app/routes/index.js

```diff
@@ -1,8 +1,9 @@
-const express = require('express');
-const controllers = require('../controllers');
-const router = express.Router();
+const express = require('express')
+const controllers = require('../controllers')
+const wrapAsync = require('../utilities/wrapAsync')
+const router = express.Router()
 
-router.route('/').get(controllers.getAllMatches);
-router.route('/:match_id').get(controllers.match_detail);
+router.route('/').get(wrapAsync(controllers.getAllMatches))
+router.route('/:match_id').get(wrapAsync(controllers.match_detail))
 
```

Comment:
All seems fine to me, but what does the addition of wrapAsync do?

Reply:
wrap async uses a call back function to help handle any errors we get

Reply:
Ah, don't worry! I saw it further down.